pwiz_tools/Skyline/Model/AuditLog/AuditLogEntry.cs

```diff
@@ -753,6 +753,7 @@ private AuditLogEntry CreateUnloggedEntry(SrmDocument doc, out bool replace)
                     .ChangeAllInfo(ImmutableList.Singleton(new MessageInfo(MessageType.log_unlogged_changes, 0)));
                 countEntry.CountEntryType = MessageType.log_unlogged_changes;
             }
+            // ReSharper disable once PossibleNullReferenceException
```

Comment:
Actually this appears to be dead code. I changed  countEntry.ChangeUndoRedo(GetUnloggedMessages(... to  countEntry?.ChangeUndoRedo(GetUnloggedMessages(...
and then I commented the whole thing out.

Reply:
It's referred to by other commented-out code. This whole area is fresh enough that my instinct says the author left the commented-out code in the belief that it might prove useful, so I've left it alone rather than deleting the two sections of currently-dead code.